quconot/implementations/mct_n_qubit_decomposition.py

```diff
@@ -31,13 +31,50 @@
 

```

Comment:
https://github.com/QuCoNot/QuCoNot/pull/8/files#diff-9e7ab7a9d95e5d45d719711c45254e390549beb32c7d7474a7df58f227ae7ee7R336-R344

this is adding classical bits for no reason. remove this.

can be replaced with
```
 mct = self.MCT(control_qubits, target_qubit, aux_qubit)
# should be done for all implementations
# TODO: solve issue with reordered qubits
 self._circuit = transpile(mct, basis_gates=["cx", "s", "h", "t", "z", "sdg", "tdg"])
```